### PR TITLE
Refactor Event Card

### LINKED ISSUE #126: Event-card refactoring
We would like the appearance of the `Event-card` component to be independent of its content - cards should look the same regardless of the length of the headings, the presence/absence of a location description, etc. Now it looks like on screenshot below

Image

NB! It's subtask #49

### PR DESCRIPTION
**Pull request by ansivgit**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [x] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description
- Add classnames library and transfer to css-module;
- Minor change in markup (classes names);
- Change styles - card height doesn't depend on it's content now;
- Remove unused styles;

## Related Tickets & Documents

- Related Issue #49 
- Closes #126 

## Screenshots, Recordings

Before
image

After
image

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

File: src/app/components/event-card/event-card.tsx [hunk 1]
```diff
@@ -13,36 +16,51 @@ export type EventCardProps = {
...
-        View details
-      </a>
+}: EventCardProps) => {
+  const dateInfo: string =
+    organizedBy.toLowerCase() === 'online'
+      ? `• ${date} • ${time}`
+      : `• ${date} • ${time} • ${type}`;
+
+  return (
+    <div className={cx('event-card')}>
```
Comment (ramzes-84) [inline src/app/components/event-card/event-card.tsx:40]: Shouldn't we use semantic tags?

File: src/app/components/event-card/event-card.tsx [hunk 2]
```diff
@@ -13,36 +16,51 @@ export type EventCardProps = {
...
+          <h4 className={cx('organized-by')}>{organizedBy}</h4>
+          <h3 className={cx('event-organization')}>{organization}</h3>
+        </div>
+        <div className={cx('about-event')}>
+          <h2 className={cx('event-title')}>{title}</h2>
+          <p className={cx('event-additional-info')}>{additionalInfo}</p>
+        </div>
+      </section>
+
+      <section className={cx('event-card__info')}>
```
Comment (ramzes-84) [inline src/app/components/event-card/event-card.tsx:53]: Do we use BEM in project? Just for clarification.

**Review by ramzes-84 — CHANGES_REQUESTED** [review]

**Comment by Quiddlee** [comment]
The gap is missing between cards

### before:
image

### after:
image

**Comment by ansivgit** [comment]
> The gap is missing between cards
> 
> ### main branch:
> image
> 
> ### your branch:
> image

**Review by ramzes-84 — APPROVED** [review]

**Review by Quiddlee — APPROVED** [review]

**Review by andron13 — APPROVED** [review]

File: src/features/home/events/events.scss [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
   &.content {
-    @include media-laptop {
+    @include media-laptop-medium {
       flex-direction: column;
-      align-items: flex-start;
+      align-items: start;
     }
 
     display: flex;
     flex-direction: row;
```
Comment (natanchik) [inline src/features/home/events/events.scss:9]: Let's remove this

File: src/features/home/events/events.scss [hunk 4]
```diff
@@ -37,17 +37,9 @@
 
       display: flex;
       flex-direction: row;
```
Comment (natanchik) [inline src/features/home/events/events.scss:39]: And this

File: src/app/components/event-card/event-card.module.scss [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+  @include media-mobile {
+    width: 100%;
+  }
+
+  width: 320px;
+  padding: 0;
+  border-radius: 12px;
+
+  &__header {
+    position: relative;
```
[the remark this answers is not in the document]
Comment (ansivgit) [reply src/app/components/event-card/event-card.module.scss:19]: We're couldn't as we have the shadow-block with `position: absolute`

[the remark this answers is not in the document]
Comment (natanchik) [reply src/app/components/event-card/event-card.module.scss:19]: ok

**Review by natanchik — APPROVED** [review]

**Pull request merged**